pages/package/package_package-xml.md

```diff
@@ -155,8 +155,8 @@ Example:
 The attribute `version` must be a valid version number as described in the [\<version\>](#version) section. In the example above it will be impossible to install this package in WoltLab Suite Core 3.1.0 Alpha 1 or higher.
 
 ### `<compatibility>`
-
 {% include callout.html content="Available since WoltLab Suite 3.1" type="info" %}
+{% include callout.html content="With the release of WoltLab Suite 5.2 the API versions were abolished. Instead an exclude to the version "6.0.0 Alpha 1" from the WSC is sufficient." type="warning" %}
```

Comment:
```suggestion
{% include callout.html content="With the release of WoltLab Suite 5.2 the API versions were abolished. Instead of using API versions packages should exclude version `6.0.0 Alpha 1` of `com.woltlab.wcf` going forward." type="warning" %}
```